### PR TITLE
Feature 2 create new account

### PR DESCRIPTION
**Pull request by MilenaMijuskovic**
Updated createAccountView and related classes to support different user types (romantic and friendship types). Created xml files for the romantic questionnaire and friendship questionnaire. Gets user inputs through createAccountView. Updated controller to make sure that it verifies that all questions have at least 1 selected answer.
!!! COMMENTED OUT SWITCH TO RECOMMENDATION VIEW AFTER USER IS CREATED. (To not cause issues, I commented out this section of code so that it does not cause errors, however, once RecommendationView has been merged, we need to uncomment so that the screens flip accordingly).
TODO: This code works. But it is VERY UNCLEAN and there is a lot of repetition of code in my createAccountView file, especially where I am getting inputs from the user. Next steps are to make code clean and shove repeated code into methods. 

edit: scroll down to read new comment

Questionnaires:
image

image

# Checklist

Please do a self-check to make sure you have satisfied all of the following before marking your PR
as ready for review (place an `x` (the letter x) between `[ ]` to mark complete).

- [x] My PR does not contain unrelated files that may cause conflicts
- [x] I have performed a self-review of my own code
- [x] I have commented my code with javadoc and perhaps in-line comments (particularly in
  hard-to-understand areas)
- [x] My changes generate no new warnings or errors
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have not violated clean architecture principles
- [x] I have opted to use the most general form of datastructures (example: use `List` instead of
  `ArrayList` wherever possible)

### CODE REVIEW

File: app/src/main/java/com/group80/uoftinder/CreateAccountView.java
Comment (rsrinivasan1) [inline app/src/main/java/com/group80/uoftinder/CreateAccountView.java:279]: Have you considered making one ```createQuestionnnaire``` method that takes in a User? You could figure out the type from the ```currentUser``` object.

File: app/src/main/java/com/group80/uoftinder/UserAccountController.java
Comment (rsrinivasan1) [inline app/src/main/java/com/group80/uoftinder/UserAccountController.java:51]: Looks good!

**Review by rsrinivasan1 — APPROVED** [review]
Cool, looks good to me! Seems like all the HashSet instances for user answers have been converted to Lists like we all talked about. I added one comment on whether you just have one method to create the questionnaire, and handle it differently based on user type.

**Review by swaner2 — APPROVED** [review]
This code is a little hard to follow along but once it is refactored and clean architecture is implemented it should be good! It's great to see progress that we're adding different kinds of users.

**Comment by MilenaMijuskovic** [comment]
Code has been made clean, no more errors are made. Branch is good to merge once merge conflicts are resolved.
[x]  I have not violated clean architecture principles

**Review by swaner2 — APPROVED** [review]
Nice to see it is clean and javadocs are added!

**Review by rsrinivasan1 — APPROVED** [review]
Everything looks great! Thanks for making this clean, Milena!

**Review by LanceLi1416 — APPROVED** [review]
Great work! Thanks for all the hard work of refectoring to make it follow clean architecture~

**Pull request merged**
